content/team/Timofei Rachkovsky.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Timofei Rachkovsky
+job_title: Design Engineer, Master's Degree student
+description: In VibroBox team Timofei is Design Engineer. He develops and tests hardware and firmware, as well as responsible for the developing signal processing modules in the spectral domain and data acquisition subsystems.
+keywords: Timofei Rachkovsky, design engineer, Master's Degree student, VibroBox, Digital Signal Processing, software testing, hardware testing
+weight: 50
+share:
+  image: img/team/Timofei Rachkovsky.jpg
+---
+In VibroBox team Timofei is Design Engineer. He develops and tests hardware and firmware, as well as responsible for the developing signal processing modules in the spectral domain and data acquisition subsystems. He is responsible for the interface between the software framework and the web part of the project. He works in a close contact with other developers to maximize the effectiveness of the final product. Also, Timofei writes user manuals for Vibrobox products.
+
+Timofei graduated from Belarusian State University of Informatics and Radioelectronics with a bachelor’s degree in Radioinformatics engineering. Timofei’s thesis theme is “Data gathering and transmission unit for the condition monitoring systems.”
```

Comment:
... graduated in 20xx from ...

... theme was "Data collection and transmission unit for condition monitoring systems."